frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx

```diff
@@ -49,14 +52,26 @@ function CellInfoSideBar({
     if (!data) return;
     const genes = Object.keys(data.marker_genes);
     navigator.clipboard.writeText(genes.join(", "));
+    track(EVENTS.WMG_FMG_COPY_GENES_CLICKED);
   }, [data]);
 
   const handleDisplayGenes = useCallback(() => {
     if (!data || !dispatch) return;
     const genes = Object.keys(data.marker_genes);
     dispatch(addSelectedGenes(genes));
+    track(EVENTS.WMG_FMG_ADD_GENES_CLICKED);
   }, [data, dispatch]);
 
+  const [hoverTime, setHoverTime] = useState({ start: -1, end: -1 });
```

Comment:
discussion: What do you think of using just one value, reportTooltipHoverEpochTime?

```ts
const [reportTooltipHoverEpochTime, setReportTooltipHoverEpochTime] = useState(0);

...

hoverStart={() => setReportTooltipHoverEpochTime(Date.now + 2 * 1000)}
hoverEnd={handleHoverEnd}

function handleHoverEnd() {
  if (Date.now >= reportTooltipHoverEpochTime) {
    track(...)
  }
}
```

Reply:
No strong feelings, except that without context this looks like some magic number stuff:
`setReportTooltipHoverEpochTime(Date.now + 2 * 1000)}`

Reply:
Yeah great call-out: maybe extract 2 * 1000 as a constant like so?

`const TOOLTIP_INTERACTION_MS = 2 * 1000`

Reply:
I'm actually going to move it to handleHoverEnd as that's more connotative with the calculation being done